include/pulse_detect.h

```diff
@@ -18,7 +8,7 @@
 
 #define PD_MAX_PULSES 1200      // Maximum number of pulses before forcing End Of Package
 #define PD_MIN_PULSES 16        // Minimum number of pulses before declaring a proper package
-#define PD_MIN_PULSE_SAMPLES 10 // Minimum number of samples in a pulse for proper detection
+#define PD_MIN_PULSE_SAMPLES 5 // Minimum number of samples in a pulse for proper detection
```

Comment:
This change I need a confirm by @zuckschwerdt if it is ok to add. I am sort of for doing that as a stop gap solution for now. If @zuckschwerdt doesn't agree the patch can go in without that change. We then maybe change PD_MIN_PULSE_SAMPLES to an option so that we can get out of the box support of this sensor.

Reply:
Yes, that number is a bit fragile. Effectivly a low pass filter for OOK and FSK demod. But it's not normalized to the sample rate. In effect it demands a minimum pulse length of 40µs. This is okay for the fastest timing of 50 µs (mostly TPMS) at 250k. But it's also the reason why IBIS beacon (30 µs) and M-Bus (10 µs, 15 µs…) need at least 1M to work.
I'll run some test to probe the whole range of e.g. 1-15 and see how fragile it really is. Ideally we should just use (and tune) the existing IIR low pass.

Reply:
Already at `9` there are two regressions (fineoffset_wh25/01/gfile005.cu8, honda_remote/01/hondacar-unlock.cu8) and at `5` a mix of 24 tests fail (alecto, current cost, oil gauge, tpms).
We really need to tune the low pass filter or something else before we can change this.

Doesn't this decoder work with higher sample rates and no change to this constant?